tests/h/search/conftest.py

```diff
@@ -16,6 +17,15 @@ def group_service(pyramid_config):
     return group_service
 
 
+@pytest.fixture(autouse=True)
+def moderation_service(pyramid_config):
```

Comment:
I saw a discussion recently about _not_ making this an `autouse` fixture because of the cost of auto-speccing `AnnotationModerationService` in all tests, including those that don't really require it. What happened to that discussion?